src/components/DataViewer/DataViewer.tsx

```diff
@@ -52,17 +52,22 @@ const formatDisplayValue = (value: valueType): string => {
 
 export interface DataViewerProps {
     data: Data;
+    fieldsToHide?: Partial<[keyof Data]>;
 }
 
-const DataViewer: React.FC<DataViewerProps> = (props) => {
+const DataViewer: React.FC<DataViewerProps> = ({ data, fieldsToHide = [] }) => {
```

Comment:
I'm not quite sure I understand what you mean by the signature? 
I just unpacked the props in order to make it more readable, I thought this was usually preferred

Reply:
Method signature :) not sure if you can really call it that here but just this whole thing `const DataViewer: React.FC<DataViewerProps> = ({ data, fieldsToHide = [] })`

Looking at some examples I think it might be the correct way, just in my head more fields == more places to update, and like I mentioned having a massive parameter list. But let's leave it for now